python/src/array.cpp

```diff
@@ -1132,6 +1132,12 @@ void init_array(py::module_& m) {
           py::kw_only(),
           "stream"_a = none,
           "See :func:`any`.")
+      .def(
+          "isposinf",
+          &mlx::core::isposinf,
```

Comment:
I don't think we need this method (numpy doesn't have it and I don't see a real need).

Reply:
so we do not need to expose this from python right?